rolling/ci-nightly-connext.yaml

```
@@ -26,7 +26,7 @@ jenkins_job_priority: 50
 jenkins_job_schedule: 15 23 * * *
 jenkins_job_timeout: 300
 jenkins_job_weight: 4
-package_selection_args: '--packages-ignore fastcdr foonathan_memory_vendor rosbag2_converter_default_plugins --packages-ignore-regex .*cyclonedds.* .*fastrtps.*'
+package_selection_args: '--packages-ignore foonathan_memory_vendor rosbag2_converter_default_plugins --packages-ignore-regex .*cyclonedds.* rmw_fastrtps.*'
```

Still want to ignore `fastrtps` itself:
```suggestion
package_selection_args: '--packages-ignore fastrtps foonathan_memory_vendor rosbag2_converter_default_plugins --packages-ignore-regex .*cyclonedds.* rmw_fastrtps.*'
```